### PR TITLE
Disable show_email_copy per default for new installations

### PR DESCRIPTION
**Pull request by zero-24**
### Summary of Changes

Disable show_email_copy per default for new installations. cc @SniperSister 

### Testing Instructions

- Take a new install from https://github.com/zero-24/joomla-cms/archive/copymail.zip
- Check the com_contact options and that "show_email_copy" is disabled
- check that per default a new form does not have show_email_copy enabled

### Expected result

per default a new form on a new install does not have show_email_copy enabled

### Actual result

per default a new form have show_email_copy enabled

### CODE REVIEW

File: templates/beez3/html/com_contact/contact/default_form.php [hunk 1]
```diff
@@ -38,7 +38,7 @@
 				<div class="control-label"><?php echo $this->form->getLabel('contact_message'); ?></div>
 				<div class="controls"><?php echo $this->form->getInput('contact_message'); ?></div>
 			</div>
-				<?php 	if ($this->params->get('show_email_copy')){ ?>
+				<?php if ($this->params->get('show_email_copy', '0')){ ?>
```
Comment (Quy) [inline templates/beez3/html/com_contact/contact/default_form.php]: Change `{` to `:`
Change `}` to `endif;`

Comment (zero-24) [reply templates/beez3/html/com_contact/contact/default_form.php]: this is a feature PR not a codestyle PR ;) I'm happy to do this in the next batches. ;)

File: components/com_contact/controllers/contact.php [hunk 2]
```diff
@@ -129,7 +129,7 @@ public function submit()
 
 		if (!$params->get('custom_reply'))
 		{
-			$sent = $this->_sendEmail($data, $contact, $params->get('show_email_copy'));
+			$sent = $this->_sendEmail($data, $contact, $params->get('show_email_copy', '0'));
```
Comment (n9iels) [inline components/com_contact/controllers/contact.php]: Should this not be a integer `0`? (Not meant as code style)

**Comment by n9iels** [comment]
I have tested this item :white_check_mark: successfully on fb144332d9a428b826504b6745216ae865020e85<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17848">issues.joomla.org/tracker/joomla-cms/17848</a>.</sub>

**Comment by brianteeman** [comment]
finally - ;)

**Comment by SniperSister** [comment]
I have tested this item :white_check_mark: successfully on fb144332d9a428b826504b6745216ae865020e85<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17848">issues.joomla.org/tracker/joomla-cms/17848</a>.</sub>

**Comment by ghost** [comment]
RTC after two successful tests.

**Pull request merged**